routes/utils.js

```diff
@@ -6,8 +6,15 @@ var async = require("async");
 var defaultProject = require("../default");
 
 function createProject(config, user, data, callback) {
-  var project = JSON.parse(JSON.stringify(data));
   var createURL = config.publishURL + "/projects";
+  var project;
+  try {
+    project = JSON.parse(JSON.stringify(data));
```

Comment:
because of the `delete project.id` below! We don't want that changing it in the caller